### PR TITLE
MLPAB-2057 Allow non-supporters to add a correspondent

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

**Review by acsauk — DISMISSED** [review]
Perfect timing with UR next week 🙏 

:shipit:

File: lang/en.json [hunk 1]
```diff
@@ -793,7 +793,7 @@
...
+    "evidenceRequiredHalfFeeEvidenceContent": "<h2 class=\"govuk-heading-m\">Supporting evidence of your income</h2><p class=\"govuk-body\">Your evidence can come from:</p><ul class=\"govuk-list govuk-list--bullet\"><li>paid employment – provide your P60 or 3 months’ consecutive wage slips from current employment</li><li>non-means-tested benefits and pensions – an official letter or notice</li><li>interest from capital, stocks, shares or bonds – statements or vouchers showing your gross income</li><li>self-employment – your most recent self-assessment tax return and HMRC tax calculation, or audited account certified by a qualified accountant</li></ul><p class=\"govuk-body\">Any evidence you send should show that the benefit or income is currently being paid to you. It should include your printed details such as your title and full name, address and postcode.</p><p class=\"govuk-body\">We do not accept bank statements.</p>",
```
Comment (hawx) [inline lang/en.json:796]: no idea why this test has decided to just pop-up, but we did have h1>h3 so good spot from it

Comment (hawx) [reply lang/en.json:796]: https://github.com/ministryofjustice/opg-modernising-lpa/commit/15618a9ba7bc488a01343ed14c2503bb2bfc8ae5 this is why

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**